src/functions/durableEntity.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpRequest, InvocationContext, output, trigger } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableClientHandler, EntityHandler } from 'durable-functions';
+
+// Replace with your own Durable entity name
+const entityName = 'Counter';
+
+const clientHandler: DurableClientHandler = async (_context: InvocationContext, req: HttpRequest, client) => {
```

Comment:
Is this -`Handler` suffix that I'm seeing for Entities and DurableClients a pattern in the new programming model? I'm checking as otherwise I'd prefer to just have `DurableClient` and `DurableEntity` as the types.

Reply:
Same with `Activity`

Reply:
Discussed offline